yt/yt/server/lib/exec_node/config.h

```diff
@@ -523,6 +523,9 @@ class TGpuManagerDynamicConfig
 
     NGpu::TGpuInfoSourceConfigPtr GpuInfoSource;
 
+    //! This option is specific for nvidia-container-runtime.
```

Comment:
...specific to?

Reply:
Created a new PR: https://github.com/ytsaurus/ytsaurus/pull/290